app/javascript/components/SponsorsTile.jsx

```diff
@@ -3,26 +3,20 @@ import PropTypes from 'prop-types';
 import Card from './Card';
 
 import Stripe from 'images/sponsors/stripe.svg';
-import Shopify from 'images/sponsors/shopify.svg';
-import Contentful from 'images/sponsors/contentful.svg';
+import Cleo from 'images/sponsors/cleo.svg';
 
 import 'stylesheets/sponsors-tile';
 
 const sponsors = [
     {
-        id: 1,
```

Comment:
I removed the IDs because the title of the company is enough to pass as a `key` prop in React.